Shared/Realm.Shared/RealmCollectionBase.cs

```diff
@@ -85,8 +89,6 @@ public int Count
             }
         }
 
-        public Schema.ObjectSchema ObjectSchema => TargetMetadata.Schema;
```

Comment:
`RealmCollectionBase<T>` is internal so removing public properties on it won't affect our users, but the same property has also been removed from the public interface, as discussed in https://github.com/realm/realm-dotnet/pull/1206#pullrequestreview-19596696.